src/software/simulated_tests/validation/world_state_validator.h

```diff
@@ -57,7 +57,7 @@ class WorldStateValidator : public ThreadedObserver<World>, public Subject<World
     void onValueReceived(World world) override;
 
     // How long to wait to receive a world, in wall-clock time
-    const Duration world_buffer_timeout = Duration::fromSeconds(1);
+    const Duration world_buffer_timeout = Duration::fromSeconds(10);
```

Comment:
Where did the 10 come from?

Reply:
These are somewhat arbitrary. I increased the value to make sure tests won't time-out unexpectedly, especially in CI if things are slower. Added a comment.